samples/python/getting-started/webfrontend/webfrontend/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "webfrontend.fullname" . }}-test-connection"
+  labels:
+    app.kubernetes.io/name: {{ include "webfrontend.name" . }}
+    helm.sh/chart: {{ include "webfrontend.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+  annotations:
+    "helm.sh/hook": test-success
```

Comment:
>"helm.sh/hook" [](start = 4, length = 14)

Just curious: Was this file generated by the default Helm3 template?

Reply:
yes, this was generated by Helm 3